otcdocstheme/theme/otcdocs/static/js/docs.js

```diff
@@ -70,7 +70,7 @@ function logABug(bugTitle, bugProject, fieldComment, fieldTags, repositoryName)
     bugChecklist + lineFeed + "-----------------------------------" + lineFeed + fieldComment +
     lineFeed + currentURL;
     document.getElementById("logABugLink1").href = bugLink;
-    document.getElementById("logABugLink2").href = bugLink;
+//    document.getElementById("logABugLink2").href = bugLink;
```

Comment:
in the openstackdocstheme there are 3 places on the page with the link. We may add it back soon, but atm just comment out since it causes JS exception